landoapi/api/transplants.py

```diff
@@ -0,0 +1,55 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""
+Transplant API
+See the OpenAPI Specification for this API in the spec/swagger.yml file.
+"""
+from connexion import problem
+from landoapi.models.transplant import (
+    Transplant,
+    TransplantNotCreatedException,
+    TransplantNotFoundException,
+    TransplantRevisionNotFoundException,
+)
+
+
+def land(revision_id, api_key=None):
+    """ API endpoint at /revisions/{id}/transplants to land revision. """
+    try:
+        transplant = Transplant.create(revision_id, api_key)
+    except TransplantRevisionNotFoundException:
+        # We could not find a matching revision.
+        return problem(
+            404,
+            'Revision not found',
+            'The requested revision does not exist',
+            type='https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404'
+        )
+    except TransplantNotCreatedException:
+        # We could not find a matching revision.
+        return problem(
+            502,
+            'Transplant not created',
+            'The requested revision does exist, but landing failed',
+            type='https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/502'
+        )
+
+    return {'id': transplant.id}, 202
+
+
+def check_status(transplant_id):
+    """ API endpoint at /revisions/{id}/transplants to list landing jobs
```

Comment:
2 things. 

This comment doesn't reflect what this function does. The comment says it lists all landing jobs for a revision, but, infact it only retrieves one landing job based on a given id.

Like I mentioned in the last PR. Having an endpoint to get an individual transplant job based on its id will serve no purpose. This is because external services (e.g. Lando UI) should not (or in the case of Lando UI literally cannot) save the transplant ids. The only way for them to query the status of a landing is by revision id. 

I propose these endpoints
`/revisions/{id}/land` - This lands a revision like you have already implemented. The change here is to remove `transplants` from the url because it doesn't really matter for external clients to know the specifics.

`/revisions/{id}/landings` - For each transplant job on that revision returns: ldap username of person that started it, date started, date finished, landing status and target repo. 

This is just a proposal, but, I think we should have a discussion about it @zalun & @mars-f.